Can't open campaign in the mobile app

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
<!-- Mandatory -->

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
1. Login in the mobile app with SurvOff
2. Navigate to Campaigns
3. Try to open a campaign
=> The app crashes and an exception appears in the logs

### Expected Behavior
<!-- Optional -->
Campaign data should be show in the app
### Screenshots
<!-- Optional -->

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device:
* SORMAS version:
* Android version/Browser:
* Server URL:
* User Role:

### Additional Information
<!-- Optional -->
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: de.symeda.sormas.app.debug, PID: 14731
    java.lang.ExceptionInInitializerError
        at org.springframework.core.SpringProperties.getProperty(SpringProperties.java:102)
        at org.springframework.expression.spel.SpelParserConfiguration.<clinit>(SpelParserConfiguration.java:35)
        at org.springframework.expression.spel.standard.SpelExpressionParser.<init>(SpelExpressionParser.java:41)
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadFragment.<init>(CampaignFormDataReadFragment.java:59)
        at java.lang.Class.newInstance(Native Method)
        at de.symeda.sormas.app.BaseFragment.newInstance(BaseFragment.java:51)
        at de.symeda.sormas.app.BaseReadFragment.newInstance(BaseReadFragment.java:60)
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadFragment.newInstance(CampaignFormDataReadFragment.java:63)
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadActivity.buildReadFragment(CampaignFormDataReadActivity.java:50)
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadActivity.buildReadFragment(CampaignFormDataReadActivity.java:32)
        at de.symeda.sormas.app.BaseReadActivity$1.accept(BaseReadActivity.java:113)
        at de.symeda.sormas.app.BaseReadActivity$1.accept(BaseReadActivity.java:109)
        at de.symeda.sormas.app.BaseReadActivity.requestRootData(BaseReadActivity.java:96)
        at de.symeda.sormas.app.BaseReadActivity.onResumeFragments(BaseReadActivity.java:109)
        at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:447)
        at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:240)
        at android.app.Activity.performResume(Activity.java:8154)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4434)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4476)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: org.apache.commons.logging.LogConfigurationException: java.lang.ClassNotFoundException: Didn't find class "org.apache.commons.logging.impl.LogFactoryImpl" on path: DexPathList[[zip file "/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/base.apk"],nativeLibraryDirectories=[/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/lib/x86, /system/lib, /system_ext/lib]] (Caused by java.lang.ClassNotFoundException: Didn't find class "org.apache.commons.logging.impl.LogFactoryImpl" on path: DexPathList[[zip file "/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/base.apk"],nativeLibraryDirectories=[/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/lib/x86, /system/lib, /system_ext/lib]])
        at org.apache.commons.logging.LogFactory.createFactory(LogFactory.java:1158)
        at org.apache.commons.logging.LogFactory$2.run(LogFactory.java:960)
        at java.security.AccessController.doPrivileged(AccessController.java:43)
        at org.apache.commons.logging.LogFactory.newFactory(LogFactory.java:957)
        at org.apache.commons.logging.LogFactory.getFactory(LogFactory.java:624)
        at org.apache.commons.logging.LogFactory.getLog(LogFactory.java:655)
        at org.springframework.core.SpringProperties.<clinit>(SpringProperties.java:51)
        at org.springframework.core.SpringProperties.getProperty(SpringProperties.java:102) 
        at org.springframework.expression.spel.SpelParserConfiguration.<clinit>(SpelParserConfiguration.java:35) 
        at org.springframework.expression.spel.standard.SpelExpressionParser.<init>(SpelExpressionParser.java:41) 
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadFragment.<init>(CampaignFormDataReadFragment.java:59) 
        at java.lang.Class.newInstance(Native Method) 
        at de.symeda.sormas.app.BaseFragment.newInstance(BaseFragment.java:51) 
        at de.symeda.sormas.app.BaseReadFragment.newInstance(BaseReadFragment.java:60) 
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadFragment.newInstance(CampaignFormDataReadFragment.java:63) 
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadActivity.buildReadFragment(CampaignFormDataReadActivity.java:50) 
        at de.symeda.sormas.app.campaign.read.CampaignFormDataReadActivity.buildReadFragment(CampaignFormDataReadActivity.java:32) 
        at de.symeda.sormas.app.BaseReadActivity$1.accept(BaseReadActivity.java:113) 
        at de.symeda.sormas.app.BaseReadActivity$1.accept(BaseReadActivity.java:109) 
        at de.symeda.sormas.app.BaseReadActivity.requestRootData(BaseReadActivity.java:96) 
        at de.symeda.sormas.app.BaseReadActivity.onResumeFragments(BaseReadActivity.java:109) 
        at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:447) 
        at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:240) 
        at android.app.Activity.performResume(Activity.java:8154) 
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4434) 
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4476) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7656) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
E/AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "org.apache.commons.logging.impl.LogFactoryImpl" on path: DexPathList[[zip file "/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/base.apk"],nativeLibraryDirectories=[/data/app/~~n4eUag2y36K5s1FwuJ9VRg==/de.symeda.sormas.app.debug-KaACBNdon1GjLq1cPR_18Q==/lib/x86, /system/lib, /system_ext/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at org.apache.commons.logging.LogFactory.createFactory(LogFactory.java:1020)
        	... 35 more
V/FA: Connection attempt already in progress
V/FA: Recording user engagement, ms: 895
D/TransportRuntime.SQLiteEventStore: Storing event with priority=HIGHEST, name=FIREBASE_CRASHLYTICS_REPORT for destination cct
D/TransportRuntime.JobInfoScheduler: Scheduling upload for context TransportContext(cct, HIGHEST, MSRodHRwczovL2NyYXNobHl0aWNzcmVwb3J0cy1wYS5nb29nbGVhcGlzLmNvbS92MS9maXJlbG9nL2xlZ2FjeS9iYXRjaGxvZ1xBSXphU3lCcnBTWVQ0RkZMMDlyZUhKaTZIOUZZZGVpU25VVE92Mk0=) with jobId=281818091 in 1000ms(Backend next call timestamp 1651670150693). Attempt 1
I/Process: Sending signal. PID: 14731 SIG: 9
Disconnected from the target VM, address: 'localhost:51919', transport: 'socket'
```

--- a/sormas-app/app/src/main/AndroidManifest.xml
+++ b/sormas-app/app/src/main/AndroidManifest.xml
@@ -26,6 +26,8 @@
         <uses-library android:name="android.test.runner"
             android:required="false" />
 
+        <uses-library android:name ="org.apache.http.legacy" android:required ="false"/>
+
         <activity android:name=".login.LoginActivity"
             android:theme="@style/LoginActivityTheme"
             android:screenOrientation="portrait"

